action.yml

```diff
@@ -25,10 +25,13 @@ inputs:
     description: 'Used to specify a package manager for caching in the default directory. Supported values: npm, yarn, pnpm.'
   cache-dependency-path:
     description: 'Used to specify the path to a dependency file: package-lock.json, yarn.lock, etc. Supports wildcards or a list of file names for caching multiple dependencies.'
+  corepack:
+    description: 'Enables Corepack which allows the use of other package managers.'
+    default: 'false'
```

Comment:
As I mentioned on https://github.com/actions/setup-node/issues/531#issuecomment-2743481990, I strongly recommend this input takes a version string instead of true / false.

If consumers want to use the latest version (at the cost of stability / security), it could look like this:
```yaml
steps:
- uses: actions/setup-node@v4
  with:
    node-version-file: .nvmrc
    corepack: latest
```